client/app/components/QueryBasedParameterInput.jsx

```diff
@@ -77,7 +77,7 @@ export default class QueryBasedParameterInput extends React.Component {
   }
 
   render() {
-    const { className, value, mode, onSelect, ...otherProps } = this.props;
+    const { className, mode, onSelect, optionFilterProp, queryId, ...otherProps } = this.props;
```

Comment:
Also, removing the `value` from here will cause it to be replaced from `otherProps`, which is causing Cypress to fail